tools/tune/crossover/crossover_build_blob.m

```diff
@@ -64,7 +64,8 @@
 %% Return current SOF ABI header
 %% Use sof-ctl to write ABI header into a file
 abifn = 'crossover_get_abi.bin';
-cmd = sprintf('sof-ctl -g %d -b -o %s', setsize, abifn);
+cmd = sprintf('./sof-ctl -g %d -b -o %s', setsize, abifn);
+cd ./../../build_tools/ctl/
```

Comment:
Not sure if this is possible for Matlab, but can we have the notion of an include file that has definitions of all the paths needed for all the matlab code. i.e. matlab code includes the header and can do a 
```
cd ${BUILD_TOOLS}/ctl/
```

Reply:
This is not needed, the sof-ctl binary should be in the developer's $PATH.

Though we do have the uncertainty that this script is not ensuring the binary is more recent than ctl.c source. The Matlab/Octave script could compile it to /tmp and run from there. The environment variables like CC and CFLAGS etc, could be retrieved from calling shell.

Reply:
Thanks for the  input .
The  problem is resolved by executing the **_sof/tools/build_tools$ sudo make install_**
following are created in the directory
Install the project...
-- Install configuration: "Debug"
-- Up-to-date: /usr/local/bin/sof-probes
-- Up-to-date: /usr/local/bin/sof-logger
-- Up-to-date: /usr/local/bin/sof-ctl

start the  octave/matlab from /usr/local/bin/ directory and change the  directory to **_sof/tools/tune/drc/example_drc_** & run, following is the  debug information confirm correct results and script execution

```
K>> fn

fn = 

    "../../topology/m4/drc_coef_default.m4"

K>> fh

fh =

     3

K>> blob_fn

blob_fn = 

    "../../ctl/drc_coef.blob"

53 4f 46 00 00 00 00 00 6c 00 00 00 00 20 01 03 
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
6c 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 00 00 00 00 00 00 00 e8 00 00 00 1e 
00 00 00 0c d3 4d 62 00 b1 c2 09 04 55 55 55 05 
50 fa 1e 00 55 60 94 00 7e 98 6a ff 83 c9 fe 01 
64 47 47 22 17 56 74 01 1c c7 71 00 77 77 77 ff 
d8 77 1f 00 05 00 00 00 00 80 43 00 d7 7d 04 00 
a0 ce 25 00 d7 7d 09 00 b1 b5 00 00 
K>> alsa_fn

alsa_fn = 

    "../../ctl/drc_coef.txt"

```

Reply:
We can close  the  issue for now  and create a draft for missing the documentation for **_`sof/tools/build_tools$ sudo make install`_**